### PR TITLE
Cleanup duplicate linitng code

### PR DESCRIPTION
**Pull request by bshewale**
As we are added pre-commit job in the same file which basically covers
all the linting checks. So removed the duplicate ones from the same
file.

### CODE REVIEW

**Comment by bshewale** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-glance-operator-master-precommit-check

**Comment by Sandeepyadav93** [comment]
As discussed,we can drop the duplicate job like golangci lint as they already run in pre-commit.

File: ci-operator/jobs/openstack-k8s-operators/glance-operator/openstack-k8s-operators-glance-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 presubmits:
   openstack-k8s-operators/glance-operator:
   - agent: kubernetes
-    always_run: false
+    always_run: true
```
Comment (viroel) [inline ci-operator/jobs/openstack-k8s-operators/glance-operator/openstack-k8s-operators-glance-operator-master-presubmits.yaml:4]: This should remains as `false`
Is the script that is making this change? In my environment, it doesn't provide this line change.
Are you using `make jobs` to fix this file?

**Review by viroel — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/openstack-k8s-operators/glance-operator/openstack-k8s-operators-glance-operator-master-presubmits.yaml (hunk 1 above)
Comment (bshewale) [reply ci-operator/jobs/openstack-k8s-operators/glance-operator/openstack-k8s-operators-glance-operator-master-presubmits.yaml]: i ran make update and that made this change.

**Review by abays — APPROVED** [review]
/lgtm

**Review by fmount — APPROVED** [review]
Looks good, thanks!

**Pull request merged**